frontend/src/components/worksheets/BundleDetail/BundleDetailSideBar.jsx

```diff
@@ -147,6 +156,11 @@ class BundleDetailSideBar extends React.Component<{
                     )}
                 </div>
                 {/** ----------------------------------------------------------------------------------------------- */}
+                <div>
+                    <ConfigLabel label='State Details:' inline={true} />
```

Comment:
Can we include this as part of the state (so it doesn't look like it's a separate field)?